web/src/components/task/task-sidebar/finish-button/finish-button.tsx

```diff
@@ -52,18 +52,17 @@ export const FinishButton: FC<TaskSidebarProps> = ({
 
     const handleFinishValidation = isSplitValidation ? onFinishSplitValidation : onFinishValidation;
 
-    let isDisabled = !isAnnotatable;
+    let isDisabled = !isAnnotatable || (isValidation && !allValidated);
     if (jobType !== 'extensive_coverage') {
         isDisabled =
             !isAnnotatable ||
             !allValidated ||
             (isValidation && !touchedPagesCount && !editedPagesCount);
     }
 
-    const tooltipContent =
-        isDisabled && allValidated && !isSplitValidation
```

Comment:
I see you've removed "&& allValidated && !isSplitValidation", before it was also combined with the "isDisabled" placed under "if". Is it needed or this logic should still be presented for the logic under "jobType !== 'extensive_coverage'"?

Reply:
I rearranged the logic so that, for non extensive coverage cases, `isDisabled` is true in the case `(isValidation && !allValidated)` is `true`. (I.e., the button is disabled if we are under validation and not all pages are validated). So now, `isDisabled` accounts for `allValidated`

From my analysis of the code, it is logical the the tooltip should only appear In the case the button is disabled. Keeping the `isSplitValidation` on the `tootipContent` calculation will make the tooltip appear in some cases the button is not disabled.

At last, the logic inside the `if` statement was not changed.